.github/workflows/sample_data.yaml

```diff
@@ -0,0 +1,38 @@
+name: "Deploy Sample Data"
+
+on:
+  pull_request_target:
+    types: [labeled]
+  workflow_dispatch:
+    inputs:
+      server:
+        description: "Staging Server"
+        type: choice
+        required: true
+        options:
+          - staging.openfoodnetwork.org.uk
+          - staging.openfoodnetwork.org.au
+          - staging.coopcircuits.fr
+      sample_data_name:
+        description: "Sample Data Name"
+        type: string
+        required: true
+
+jobs:
+  deploy_sample_data:
+    runs-on: ubuntu-latest
+    steps:
+      - name: "Check user has write access"
+        uses: "lannonbr/repo-permission-check-action@2.0.2"
+        with:
+          permission: "write"
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Run sample data script
+        if: success()
+        run: |
+          sudo systemctl stop puma sidekiq
+          ~/apps/openfoodnetwork/current/script/ci/save-staging-baseline.sh
```

Comment:
I'm sorry but this doesn't work.

This script is executed on the Github Actions machine. It's not executed on one of the staging servers. In theory, we could execute commands via `ssh` on the staging servers but at the moment they don't allow that. We would need to authorize an SSH key stored in Github to perform these commands. And I'm not sure we want that.

Reply:
> This script is executed on the Github Actions machine

Uff, I obviously I missed that as well. Thanks for pointing that out @mkllnk.

> What we need is for this script to be saved on the server. Then it's pretty trivial to log in and run it.

Ok, I'll have a second look at this. I'm not sure though I fully grasp the complexity here - I'm looking out for this Slack [thread](https://openfoodnetwork.slack.com/archives/C02TZ6X00/p1738625108292989?thread_ts=1738611204.859169&cid=C02TZ6X00), and then proceed. Thanks for your help and reviews.

Reply:
Sorry I wasn't very prescriptive. I'm not sure if Maikel has concerns about this approach. 
I suggest the next step is to create a new script in ofn-install. I had a look.. it's not easy to explain.. I'll try it..

Reply:
Maikel what do you think of this? https://github.com/openfoodfoundation/ofn-install/pull/994

Reply:
Yes, all possible. I was just cautious because it's not quick and easy. But if there's enough appetite for this feature then sure we can do it.